backend/api.test/EventHandlers/TestMissionEventHandler.cs

```diff
@@ -194,14 +196,14 @@ public async void NewMissionIsStartedWhenRobotBecomesAvailable()
         }
 
         [Fact]
-        public async void NoMissionIsStartedIfQueueIsEmptyWhenRobotBecomesAvailable()
+        public async void ReturnToHomeMissionIsStartedIfQueueIsEmptyWhenRobotBecomesAvailable()
```

Comment:
Is this test testing what the name suggests?

Seems to be missing a check that a return to home mission actually is started when the robot becomes available and no missions is pending.

Consider making a new issue for this

Reply:
See #1445